privacy_estimates/binomial_proportion.py

```diff
@@ -59,6 +60,33 @@ def compute_eps_lo_hi(count: AttackResults, delta: float, alpha: float, method:
     assert (fpr_l <= fpr and fpr <= fpr_r)
     assert (fnr_l <= fnr and fnr <= fnr_r)
 
+    return fpr_l, fpr_r, fnr_l, fnr_r
```

Comment:
I think most of the other code assumes fnr, fpr (in that order). We could stick to that convention here as well